application/service/services.go

```diff
@@ -52,6 +52,7 @@ type RoleManagementService interface {
 	ListByResourceAndRoleName(ctx context.Context, resourceID string, roleName string) ([]rolerepo.IdentityRole, error)
 	Assign(ctx context.Context, assignedBy uuid.UUID, roleAssignments map[string][]uuid.UUID, resourceID string, appendToExistingRoles bool) error
 	ForceAssign(ctx context.Context, assignedTo uuid.UUID, roleName string, res resource.Resource) error
+	DeleteRoleAssignments(ctx context.Context, byIdentity uuid.UUID, forIdentities []uuid.UUID, resourceID string) error
```

Comment:
Can we call this `RevokeResourceRoles()` instead?

Reply:
Also, I'm wondering if we could standardize naming a little more, in other places we use `currentUser` to refer to the `identity` object for the currently-authenticated user, could we perhaps use `currentUserID` here instead of `byIdentity`, and then instead of `forIdentities` just call the parameter `identities`?